app/lib/usecases/runs/GetAllRuns.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * @license
+ * Copyright CERN and copyright holders of ALICE O2. This software is
+ * distributed under the terms of the GNU General Public License v3 (GPL
+ * Version 3), copied verbatim in the file "COPYING".
+ *
+ * See http://alice-o2.web.cern.ch/license for full licensing information.
+ *
+ * In applying this license CERN does not waive the privileges and immunities
+ * granted to it by virtue of its status as an Intergovernmental Organization
+ * or submit itself to any jurisdiction.
+ */
+
+const {
+    repositories: {
+        RunRepository,
+    },
+} = require('../../database').databaseManager;
+const { runAdapter } = require('../../database/adapters');
+
+/**
+ * Executes this use case.
+ *
+ * @param {Object} dto The GetRunsDto containing all data.
+ * @returns {Promise<Run[]>} Promise object represents the result of this use case.
+ */
+const GetRunUsecase = async () => {
```

Comment:
As I presented you, the service layer must be independent of the API, so it should not know about any DTO.
The bookkeeping usecases is the old way of doing and should not be reproduce. 
If you need an example please have a look at DplProcessService.